### PR TITLE
[16.0] [ADD] partner_brand

### PR DESCRIPTION
**Pull request by bealdav**
Allow to set brand to partners allowing to group/filter partners by brand

A brand, here, could be Carrefour, Wallmart, etc.

It also display brand image in form view and flag for partner with brand in kanban

brand_form
brand_tree
brand_kanban

brand

> I've made this module as custom one, do you think it can be useful here or with feature inside `brand` module

cc @sbejaoui @Rad0van @GSLabIt @andreampiovesana @ThomasBinsfeld 

Other idea for this concept, but to implement in an other module, i.e. product_brand_pricelist :

- define a pricelist to the partner (brand partner)
- create a pricelist on each partner based on previous pricelist
- or any suggestion ?

### CODE REVIEW

File: partner_brand/models/partner.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    brand_id = fields.Many2one(comodel_name="res.brand")
+    brand_logo = fields.Image(
+        comodel_name="res.partner",
```
Comment (qtheuret) [inline partner_brand/models/partner.py]: `fields.Image` is not a related fields so we don't need this statement.

**Review by qtheuret — CHANGES_REQUESTED** [review]
`fields.Image` is not a related fields so we don't need this statement.

File: partner_brand/models/partner.py (hunk 1 above)
Comment (bealdav) [reply partner_brand/models/partner.py]: Oops

**Review by qtheuret — APPROVED** [review]

**Review by PaulGoubert — APPROVED** [review]

File: partner_brand/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Partner Brand Custom",
```
Comment (sbejaoui) [inline partner_brand/__manifest__.py]:
```suggestion
    "name": "Partner Brand",
```

**Review by sbejaoui — APPROVED** [review]
LGTM, code review,

Just check the module name please

**Comment by bealdav** [comment]
@sbejaoui fixed. Could you merge ?

Thanks

**Comment by bosd** [comment]
ping @sbejaoui :)

**Pull request merged**